src/api/app/models/update_notification_events.rb

```diff
@@ -68,10 +67,11 @@ def perform
           BackendInfo.lastnotification_nr = Integer(@last['next'])
           return
         end
-        limit_reached = @last['limit_reached'].present?
 
         create_events
       end
-    end while limit_reached
```

Comment:
@openSUSE/open-build-service Is this a bug or just bad wording, or am I missing something? I would expect that if the limit is reached it stops the loop. Instead it continues only if it got reached? 😕